db/migrate/20170529142557_unify_container_definition_and_container.rb

```diff
@@ -0,0 +1,159 @@
+class UnifyContainerDefinitionAndContainer < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+    has_many :container_port_configs
```

Comment:
@zeari we need to add `class_name` for all relations:
`:class_name => 'UnifyContainerDefinitionAndContainer::ContainerPortConfig'`
because without it, it will not take models defined in migrations.